content/books/deathworlders/chapter-22-warhorse-pt1-first-year.md

```diff
@@ -1,928 +1,566 @@
----
-title: "Chapter 22: Warhorse Part 1—The First Year"
-date: 2015-09-18
-aliases:
-  - /book/deathworlders/chapter-22-warhorse
----
-
-**Date Point: 4y 10m AV**  
+--- title: "Chapter 22: Warhorse Part 1—The First Year" date: 2015-09-18 aliases:   - /book/deathworlders/chapter-22-warhorse ---
```

Comment:
The metadata block is...pretty thoroughly broken, in all the files. It's a YAML front matter block encased in three dashes. It has to be formatted **precisely**  and will not tolerate this.